blazor/image-editor/open-save.md

```diff
@@ -79,6 +258,123 @@ In the following example, the [`ExportAsync`](https://help.syncfusion.com/cr/bla
+
+### Save the image as blob
+
+To save an image as a blob, use the [`GetImageDataAsync`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.ImageEditor.SfImageEditor.html#Syncfusion_Blazor_ImageEditor_SfImageEditor_GetImageDataAsync) method of the editor to retrieve the image data and convert it into a blob. You can then invoke the open method on the Syncfusion Image Editor instance to load this byte array into the editor. The resulting byte array can be stored in a database for data management and maintenance.
+
+```cshtml
+@using Syncfusion.Blazor.ImageEditor
+@using Syncfusion.Blazor.Buttons
+@inject IJSRuntime JS
+
+<div style="padding-bottom: 15px">
+    <SfButton OnClick="SaveAsync">Save Blob</SfButton>
+</div>
+
+<SfImageEditor @ref="ImageEditor" Height="400">
+    <ImageEditorEvents Created="OpenAsync"></ImageEditorEvents>
+</SfImageEditor>
+
+@code {
+    private SfImageEditor ImageEditor;
+    private string blobUrl;
+
+    private async Task OpenAsync() 
+    { 
+        await ImageEditor.OpenAsync("nature.png"); 
+    }
+
+    private async Task SaveAsync()
+    {
+        var imageData = await ImageEditor.GetImageDataAsync();
+        blobUrl = await CreateBlobUrl(imageData);
+    }
+
+    private async Task<string> CreateBlobUrl(byte[] imageData)
+    {
+        var base64String = Convert.ToBase64String(imageData);
+        var jsCode = $@"
+            return new Promise((resolve) => {{
+                const byteCharacters = atob('{base64String}');
+                const byteNumbers = new Uint8Array(byteCharacters.length);
+                for (let i = 0; i < byteCharacters.length; i++) {{
+                    byteNumbers[i] = byteCharacters.charCodeAt(i);
+                }}
+                const blob = new Blob([byteNumbers], {{type: 'image/png'}});
+                resolve(URL.createObjectURL(blob));
+            }});
+        ";
+
+        return await JS.InvokeAsync<string>("eval", jsCode);
+    }
+}
+```
+### Add Watermarks while saving the image 
+
+User can utilize the ‘[`BeforeSave`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.ImageEditor.SaveEventArgs.html)’ event, which triggers just before the image is downloaded, to apply a text annotation as a watermark. After the image is downloaded, the ‘[`SaveEventArgs`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.ImageEditor.SaveEventArgs.html)’ event is triggered, allowing you to remove the watermark using the ‘[`DeleteShapeAsync`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.ImageEditor.SfImageEditor.html#Syncfusion_Blazor_ImageEditor_SfImageEditor_DeleteShapeAsync_System_String_)’ method. This ensures that the watermark is only visible in the downloaded image and not in the editor.
```

Comment:
After the image is downloaded, the ‘[`SaveEventArgs`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.ImageEditor.SaveEventArgs.html)’ event is triggered - SaveEventArgs event ??

Reply:
Since the Blazor Image Editor does not have a "Saved" event, I am removing the corresponding code snippet.